AnimalCompetition/Swords.java

```diff
@@ -0,0 +1,14 @@
+public abstract class Swords {
+
+  public abstract String type();
+  
+  public abstract int durability();
+
+  public abstract int price();
```

Comment:
I don't know what you're going to use `price` for but looks like Animals and Swords have a price... this would be a good use case for an Interface to hold the price (and anthing else related to buying and selling?).  Then Animals and Swords (and anything else in the future) can implement this interface and you can consolidate logic to work with price.